packages/medusa/src/services/cart.ts

```diff
@@ -95,6 +95,10 @@ type InjectedDependencies = {
   priceSelectionStrategy: IPriceSelectionStrategy
   productVariantInventoryService: ProductVariantInventoryService
   pricingService: PricingService
+  remoteQuery: (
```

Comment:
question: Should we have a type for this? I imagine we'll use this more going forward

Reply:
I've added `RemoteQueryQuery` type to `types`